hack/vendor.sh

```diff
@@ -44,7 +44,9 @@ clone git github.com/boltdb/bolt v1.1.0
 clone git github.com/docker/distribution 568bf038af6d65b376165d02886b1c7fcaef1f61
 clone git github.com/vbatts/tar-split v0.9.11
 
-clone git github.com/docker/notary 45de2828b5e0083bfb4e9a5a781eddb05e2ef9d0
+# get desired notary commit, might also need to be updated in Dockerfile
+clone git github.com/docker/notary ffca6fb522fc0dc9d582c848b20746e9cda9ca87
```

Comment:
Just as an FYI, we're pushing more and more to get our dependencies to tag, which will make it easier for us to look into a changelog to understand changes as a we bump dependencies versions.